app/ldap.js

```diff
@@ -564,6 +564,11 @@ exports.addGroup = function (group, cb) {
   });
 };
 
+// check if LDAP is disabled
+exports.isDisabled = function() {
+  return (!process.env.NODE_ENV || 'development' === process.env.NODE_ENV) && 'true' !== process.env.LDAP;
```

Comment:
This is not mistake? Lets check your expression step by step:
```
!'development' // always will be false
```
! (NOT) operator has higher priority, that ===
```
false === process.env.NODE_ENV // will be true, only if process.env.NODE_ENV equal boolaen 'true
```
So, this is not we expect.
Have you run tests?